src/Libraries/CoreNodeModels/Input/FileSystem.cs

```diff
@@ -317,6 +317,7 @@ public override void Dispose()
     [NodeCategory("Core.File")]
     [NodeDescription("DirectoryObjectNodeDescription",typeof(Resources))]
     [NodeSearchTags("DirectoryPathSearchTags", typeof(Resources))]
+    [OutPortTypes("object")]
```

Comment:
shouldn't this be `var`?

Reply:
@mjkkirschner I updated to be consistent with file from path, see PR description. I think anyway `object` seem to be made up, but I am not sure how many instances of that so I only make it consistent for now

Reply:
but looking at the `buildOutputAST` method for fileSystemObject<T> - it looks like the correct return type is `T`

Reply:
Sure, I removed type `object` and updated for three nodes